bridge.js

```diff
@@ -994,6 +1000,24 @@ Receiver.prototype = {
 Emitter(Receiver.prototype);
 
 /**
+ * Error object can't be sent via
+ * .postMessage() so we have to
+ * serialize them into an error-like
+ * Object that can be sent.
+ *
+ * @param  {*} err
+ * @return {Object|*}
+ */
+function serializeError(err) {
+  switch (err && err.constructor.name) {
+    case 'Error': return { message: err.message };
+    case 'DOMError': return { message: err.message, name: err.name };
+    case 'DOMException': return { message: err.toString() };
+    default: return err;
```

Comment:
Hmmm perhaps this was just for this particular case. It's tricky to inspect these Error objects when they're inside Workers. Are you still in favour of keeping the Error object structure the same then?